In [6]:
import pandas as pd
import numpy as np
TS = pd.read_csv("City Traffic Signs.csv")
TS.head()
TS["neighborhood"]
c = TS["neighborhood"].value_counts()
c
#TSmacs = c.max()
#TSF = c.divide(TSmacs).mul(0.1)
#TSF.head()
#TSF.columns = ["neighborhood", "values"]
#TSF.head()

South Side Flats             3672
Bloomfield                   3627
Central Business District    3491
Shadyside                    3370
Squirrel Hill South          3281
                             ... 
Ridgemont                      62
Chartiers City                 44
Glen Hazel                     38
Arlington Heights              21
Mount Oliver Borough           17
Name: neighborhood, Length: 91, dtype: int64

Introduction: The project is about the occurance percentage of traffic signs in every district in Pittsburgh. We calcualte the quantity of traffic signs in each district, and divide it by maximum quantity of traffic sign. Finally, we multiply each value by 0.1 to calculate the percentage of occurance of traffic signs in each district in Pittsburgh.
The Metric: We set the occurance rate of traffic signs in each district in Pittsburgh as the metric. We are using the data about each traffic signs in each district in the "City Traffic Sign" table. And we measure occurance times of traffic signs in each district and the percentage of it by dividing maximum quantity of traffic sign in one district.
The Best Neighborhood: The South Side Flats is the best neighborhood. From the calculation above, we find that the South Side Flats district has the largest percentage of occurance of traffic signs among all regions. Due to its the highest percentage of occurance of traffic signs, the South Side Flats is much safer than other region.The more traffic signs there are, the much more carefully and cautiously drivers will be. The speed of their cars would be slower by the traffic signs as the reminder. For example, if there are many sidewalks and stop signs in this region, drivers need to make their traffic slow down and stop frequently to check pedestrians' safe, even if there will be many traffics. Thus, the risk of being hurt for pedestrains will definitely decreases. For South Side Flats, it has 3672 traffic signs; it is deservedly the safest area.
Conclusion:.